app/src/main/java/com/example/bigowlapp/activity/MonitoringGroupPageActivity.java

```diff
@@ -124,47 +128,60 @@ public boolean onContextItemSelected(MenuItem item) {
         String[] menuItems = getResources().getStringArray(R.array.context_menu);
         String menuItemName = menuItems[menuItemIndex];
 
-        switch (menuItemName) {
-            case "View Profile":
-                break;
-            case "Remove": {
-                mGroupPageViewModel.removeUserFromGroup(contextMenuSelectedUser);
-                mUsers.remove(contextMenuSelectedUser);
-                resetUsersListViewAdapter();
-            }
+        if ("Remove".equals(menuItemName)) {
```

Comment:
Switch the order. It should be menuItemName.equals("Remove");

Reply:
Thanks for pointing it out!